eclipse-theia/theia
# upgrade NPM builtin version to 1.44.2

**upgrade NPM builtin version to 1.44.2**

- The vs code built in extension `npm` does not work with Theia's plugin
extension due to not being able to find the constructor of `Task2`. This
pull request addes the missing constructor, and upgrades the version of
`npm` in Theia.
- fixed #7779 


**added `detail` to the task config schema**

- In VS Code a `detail` property can be added to the contributed and configured tasks. The content of that `detail` property is displayed as part of the task quick open menu items. With changes in this commit, `detail` is added to the task config schema.


Signed-off-by: Liang Huang <lhuang4@ualberta.ca>


#### How to test

- Start a workspace where scripts are defined in `package.json`.
- Check if all scripts are available in the NPM VIEW, and if the scripts can be started from the view.

#### Review checklist
